src/core/execute.c

```diff
@@ -1195,7 +1195,7 @@ static int setup_pam(
 
         pam_code = pam_setcred(handle, PAM_ESTABLISH_CRED | flags);
         if (pam_code != PAM_SUCCESS)
```

Comment:
I caught up on the discussion on the other PR and this would ignore all non-success codes. Are you sure this shouldn't be more selective about which codes to ignore like only skipping `PAM_CRED_UNAVAIL` mentioned in https://github.com/systemd/systemd/pull/14569#issuecomment-574528389?

Reply:
Yeah, that's a good point.

Reply:
It's not the error I am seeing for this user. Not sure where that specific error code is coming from?

PAM modules can return virtually anything. If we intend to be compatible with whatever people throw at us we can only ignore the errors wholesale.

Reply:
Right. The error might be `PAM_CRED_ERR`. This is not clear how stable this is, so indeed, ignoring all errors seems appropriate.

OK, let's merge.